src/main/webapp/join.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Plant Battles</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <script src="script.js"></script>
+    <script src="https://trefle.io/api/plants?q=rosemary&token=VU4zZmx5SnF3dmxVYTVsRlBTNGlqZz09"></script>
+  </head>
+  <body>
+      <nav class="navbar navbar-expand-md fixed-top nav">
+        <a class="navbar-brand" href="index.html">Plant Battles</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarCollapse">
+          <ul class="navbar-nav mr-sm-2">
+            <li class="nav-item dropdown mt-2 mt-md-0">
+                <a class="nav-link dropdown-toggle" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                Menu
+                </a>
+                <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+                <a class="dropdown-item" href="#">Profile</a>
+                <a class="dropdown-item" href="join.html">Join a Game</a>
+                <a class="dropdown-item" href="start.html">Start a Game</a>
+                <div class="dropdown-divider"></div>
+                <a class="dropdown-item" href="#">Logout</a>
+                </div>
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <br>
```

Comment:
instead of adding all these br, u can just change the margin between the first div and the header. I think margin-top:50px would work. I can do this in the CSS too.

Reply:
okay cool I was struggling with the margins before don't know if you can tell but it worked to change the margin like that (I did 200 pixels instead) and it made the background work a little better